build/package/LICENSE

```diff
@@ -1788,3 +1788,370 @@ licenses.
     software is granted. This software is provided "as is" without
     express or implied warranty, and with no claim as to its
     suitability for any purpose.
+
+- @protobufjs/aspromise@<VERSION>
+- @protobufjs/base64@<VERSION>
+- @protobufjs/codegen@<VERSION>
+- @protobufjs/eventemitter@<VERSION>
+- @protobufjs/fetch@<VERSION>
+- @protobufjs/float@<VERSION>
+- @protobufjs/inquire@<VERSION>
+- @protobufjs/path@<VERSION>
+- @protobufjs/pool@<VERSION>
+- @protobufjs/utf8@<VERSION>
+- protobufjs@<VERSION>
+  This product bundles the above 'protobufjs' node packages.
+  These packages are available under the BSD-3-Clause License:
+    This license applies to all parts of protobuf.js except those files
+    either explicitly including or referencing a different license or
+    located in a directory containing a different LICENSE file.
+
+    ---
+
+    Copyright (c) 2016, Daniel Wirtz  All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are
+    met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of its author, nor the names of its contributors
+      may be used to endorse or promote products derived from this software
+      without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+    A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+    OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+    SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+    LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+    DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+    THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+    ---
+
+    Code generated by the command line utilities is owned by the owner
+    of the input file used when generating it. This code is not
+    standalone and requires a support library to be linked with it. This
+    support library is itself covered by the above license.
+
+- google-protobuf@<VERSION>
```

Comment:
Same here and for other dependencies, we can just list the extension.js file here.